proto/plan_common.proto

```diff
@@ -215,6 +224,10 @@ message AdditionalColumn {
     AdditionalColumnHeader header_inner = 5;
     AdditionalColumnFilename filename = 6;
     AdditionalColumnHeaders headers = 7;
+    AdditionalDatabaseName database_name = 8;
```

Comment:
Since they're all empty messages, can we use the same message type instead? Or directly use `google.protobuf.Empty`?

Reply:
IIRC, prost has some trouble with google well-known types. 
& I use message structs for future extensibility on the column def, such as 

```
message AdditionalColumnHeader {
  string inner_field = 1;
  data.DataType data_type = 2;
}
```

we won't want to handle the changes inside the catalog in the future.

Reply:
```
message AdditionalColumn {
  oneof column_type {
    AdditionalColumnKey key = 1;
    AdditionalColumnTimestamp timestamp = 2;
    AdditionalColumnPartition partition = 3;
    AdditionalColumnOffset offset = 4;
    AdditionalColumnHeader header_inner = 5;
    AdditionalColumnFilename filename = 6;
    AdditionalColumnHeaders headers = 7;
  }
}
```
I think we need different message type for the `oneof`.